examples/asset_licences.md

```diff
@@ -0,0 +1,9 @@
+The "Tuffy" font is public domain, was made by Thatcher Ulrich, and was accessed from here: http://www.publicdomainfiles.com/show_file.php?id=13486218041168
+The loop doodle_pop.ogg is public domain and was made by MrZeusTheCoder.
```

Comment:
While we know now that you made this sound specifically for this, a few years down the line, it won't necessarily be clear who MrZeusTheCoder is and how the file got into the public domain. Have you published this clip somewhere that we could link to?
Also, could you export it at a lower bitrate than 1411kbps, trying not to enlarge the repository size with binary data too much.

Reply:
Yep. I will look into doing that. I didn't really make it clear that I wasn't even sure if you guys wanted to keep the file, so I didn't put much work into it.

I very much will do that. Binaries, :disappointed: .